### ISSUE TITLE
Any example of creating 2d mixer state manually?

### ISSUE THREAD

**Issue by drakeguo** [KybernetikGames/animancer#349]
I need to create a 2d Cartesian mixer that is equivalent to a blend tree in an animator controller in script. I tried to create a `CartesianMixerState` using code like this
```
        public CartesianMixerState CreateCartesianMixerState()
        {
            CartesianMixerState mixer = new CartesianMixerState();
            mixer.Speed = 1.0f;
            mixer.SetDebugName(stateName);
            foreach (var animStateDesc in animStateDescs)
            {
                mixer.Add(animStateDesc.motion, animStateDesc.position);
            }
            return mixer;
        }
```

then I tried to play this `CartesianMixerState` using `AnimancerComponent.Play` but it didn't play the animations as I expected. I can see this state in the inspector window of AnimancerComponent. But it seemed this state is not playing at all.

I saw there is a example of manually creating `LinearMixerState` and using it in the [Documentation](https://kybernetik.com.au/animancer/docs/manual/blending/mixers/creation/#manual-creation). I wonder if `CartesianMixerState` can be used in this way as well. 

Thanks.

**Comment by KybernetikGames** [comment]
That code looks like it should work fine. What does the Inspector look like?

**Comment by drakeguo** [comment]
normally, if I play the blend tree in the AnimatorController, there is a green bar indicating that the animation or AnimatorController is being played.
![1715253233073_7E15C47C-0148-4fa8-B67F-6E025FD6DE19](https://github.com/KybernetikGames/animancer/assets/8299334/48f1abcf-2e4e-4a25-925e-d9150566cde1)

But when I try to play `CartesianMixerState` created in script, or `MixerTransition2D` created in inspector. there is no green bar, and the animation is not playing at all. 
![1715253132507_83177709-1358-47bb-8E66-23880268E5DC](https://github.com/KybernetikGames/animancer/assets/8299334/6aa4d543-fffc-47ba-8d27-ee5e66f8f8b2)
the character will fade into its T-Pose, without playing animation. then when you play another animation, it will fade into the new animation from this T-Pose. so it seems that the transition kind of works, but MixerState seems to be outputing a T-Pose instead of  an animation.

**Comment by KybernetikGames** [comment]
Are you sure you're playing it and not just creating it? I can't think of any reason for that to not work, I have a bunch of unit tests which create mixers just like that.

If you can't figure it out, feel free to send a minimal reproduction project to animancer@kybernetik.com.au so I can take a look at it.

**Comment by drakeguo** [comment]
I found where the problem seems to be. Probably due to a bug in older version of Animancer. I have to call `state?.RecreatePlayable()` for `ClipState`, otherwise the animation will not be played from its start. but if I called `state?.RecreatePlayable()` for `CartesianMixerState` it will freeze the animation. Not calling `state?.RecreatePlayable()` solved my problem.

Thank you very much for the information.
